website/src/layouts/base/BaseLayout.astro

```diff
@@ -1,6 +1,10 @@
 ---
+// eslint-disable-next-line import/no-deprecated -- "Parse errors in imported module '@auth/core/types': parser.parse is not a function (undefined:undefined)"
+import { ToastContainer } from 'react-toastify';
+
 import Footer from './Footer.astro';
 import Header from './Header.astro';
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
Ah, I didn't know that you can also import it here. Then we could actually also move the CSS imports from the script tag below to here.

Reply:
I took it from loculus. :)

I can try to move it down.

Reply:
I meant moving the others to this place here, if that also works.